### DISCUSSION TITLE
Revox, please see this.

### DISCUSSION THREAD

**Discussion by SpyrosTheBoss** [revoxhere/duino-coin#1667] [General]
So before 1 year you or your mods banned my account OK? I think the only TOS I violated was to have multiple accounts but. I read that in the Duino-Coin site. I made multiple account on your coin Magi website to abuse exchange. My mistakes were honestly preety bad and I've put my mind together, so please... unban me. My username is: spilol2

**Comment by Jonny11bonk** [comment]
Hi,
This is not the place for a ban appeal. Bans are permanent. So that you know, you've broken part of the Terms you accepted upon registration. We do NOT want any rule breakers on our platform. I hope that everything is now clear to you.

Sincerely,
Duino-Coin moderator Jonny11.

**Reply by SpyrosTheBoss** [reply]
If only you guys responded to ANY of the emails I sent I wouldn't come here. You didn't even respond saying no at the emails. You basically ghosted me guys, I even tried from different email addresses but still nothing.

**Reply by Jonny11bonk** [reply]
Contacting either the owner or the staff team for a ban appeal is not just a waste of time but it's not even possible to appeal the ban. Unfortunately, this is not my case to deal with as I don't even know what exactly happened and I can't provide you with any more info about what happened over time.

**Reply by SpyrosTheBoss** [reply]
Yo its been like almost 3 years so for old time's sake, kys (^///^)
